### DISCUSSION TITLE
Explore Longhorn volume

### DISCUSSION THREAD

**Discussion by suikast42** [longhorn/longhorn#4167] [Q&A]
Is there a tool for exploring the lonhorn volumes with ls or something like that? 

I see there is a img file in my volume directory butr don't now how to explore it

**Comment by derekbit** [comment]
You can access the files in the volume via the application pod.
```
kubectl exec -it <app volume name> -- ls <mountpoint of the longhorn volume> 
```

**Reply by suikast42** [reply]
Thanks, 

what I mean was a litte bit more low level solution on the storage host. 

1. Idendtiyfy the PVC and Host over Longhorn UI
2. lsblk -f for find the block device
3. do a mount on this block device

That solves my problem

**Reply by derekbit** [reply]
If the volume is attached, you can use `kubectl -n longhorn-system get volumes -o wide` to determine which node the volume is mounted on.
Then, on the node,  find the mountpoint by `mount | grep <pvc name>`.

BTW, what's your use case?

**Reply by suikast42** [reply]
> BTW, what's your use case?

I am evaluating some disaster recovery scenerios.

**Reply by derekbit** [reply]
This will help you rescue the data from one of the replicas.
https://longhorn.io/docs/1.3.0/advanced-resources/data-recovery/export-from-replica/

**Reply by suikast42** [reply]
That's lokking good thanks.
